### PR TITLE
Privacy policy and Terms of Service

### PR DESCRIPTION
**Pull request by BNMoyers**
# Description
Privacy Policy and ToS copy implemented as simple React Components with basic html formatting, rendered on dedicated pages. 
Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [x] New feature (non-breaking change which adds functionality)

## Change Status

- [x] Complete, tested, ready to review and merge
- [x] Complete, but not tested (may need new tests)
- [x] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] There are no merge conflicts

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
added graig's comments

**Review by ghost — COMMENTED** [review]
start of writing down privacy policy.

**Review by ghost — COMMENTED** [review]
Well formatted privacy policy.

File: src/app/components/policies/privacy-policy.component.tsx [hunk 1]
```diff
@@ -0,0 +1,268 @@
+import React from 'react';
+
+const PrivacyPolicy: <React className="FC"></React> = () => {
```
Comment (EricBanker12) [inline src/app/components/policies/privacy-policy.component.tsx]: Autocomplete messed up this line. Should be:
```
const PrivacyPolicy: React.FC = () => {
```

File: src/app/pages/tos.page.tsx [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+
+const ToSPage: React.FC = () => {
+    return (
+        <>
+            <ToS />
+        </>
+    );
+};
+
+export { PrivacyPage };
```
Comment (EricBanker12) [inline src/app/pages/tos.page.tsx]: Should be `export { ToSPage };`

**Review by EricBanker12 — CHANGES_REQUESTED** [review]
There are a couple minor typos that got through and some very basic testing (does it render) testing should be added.

**Review by EricBanker12 — APPROVED** [review]
Some of the component and page naming could be better, but it's good enough for my approval.

**Review by wSedlacek — APPROVED** [review]
There is some cleanup to do with white spaces and what not but this can be cleaned as we go.
Good work Britney!

**Pull request merged**